### PR TITLE
Roll zlib and emsdk to 3.1.32

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
This will prevent some breakages with https://github.com/flutter/flutter/issues/124971

### CODE REVIEW

File: tools/activate_emsdk.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 EMSDK_PATH = os.path.join(EMSDK_ROOT, 'emsdk.py')
 
 # See lib/web_ui/README.md for instructions on updating the EMSDK version.
-EMSDK_VERSION = '3.1.3'
+EMSDK_VERSION = '3.1.36'
```
Comment (aam) [inline tools/activate_emsdk.py:19]: Skia seems to be on 3.1.15 and https://github.com/flutter/engine/blob/main/lib/web_ui/README.md#upgrading-the-emscripten-sdk-for-the-canvaskit-build advises to keep in sync with Skia

Comment (eyebrowsoffire) [reply tools/activate_emsdk.py]: After some investigation, 3.1.15 has a bug that breaks skwasm compilation. 3.1.32 seems to work for both CanvasKit and skwasm, so let's try using that one.

Comment (aam) [reply tools/activate_emsdk.py]: Skia is still on 3.1.15, right? Is it not a concern?

Comment (eyebrowsoffire) [reply tools/activate_emsdk.py]: It's not an actual concern. Keeping the version of the emsdk exactly in sync is not necessary.

**Comment by chinmaygarde** [comment]
From Triage: zlib was rolled a couple of days ago. Can we make this patch roll just emsdk please?

**Comment by eyebrowsoffire** [comment]
emsdk was already rolled in a different change since I made this PR, so this can just be closed.

**Pull request closed without merging**